static_websites/practice/todo-js/script/todo.js

```diff
@@ -4,3 +4,12 @@ let todos = [
   {id: 3, text: "Get oil change", complete: false},
   {id: 4, text: "Write thank-you notes", complete: false},
 ];
+
+function displayTodosText() {
```

Comment:
how about calling this function `someOtherName()`

Reply:
You got it!